build/Dockerfile

```diff
@@ -1,8 +1,20 @@
+FROM openshift/origin-release:golang-1.10
```

Comment:
Why not use the official docker images from https://hub.docker.com/_/golang ?

Reply:
Because this one is (nearly) identical with our downstream one, thus we can catch issues early on

Reply:
OK, that's a good enough reason for me  :-)